ci-operator/step-registry/openshift-qe/network-perf/openshift-qe-network-perf-commands.sh

```diff
@@ -4,6 +4,18 @@ set -o nounset
 set -o pipefail
 set -x
 cat /etc/os-release
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
so if this file exists in the directory we will apply the proxy? Could we maybe have an env var to also enable this? maybe `DISCONNECTED_PROXY` which for each job we can set true/false?

Reply:
Yes, if this file exists in the directory, it will apply the proxy. 
Regarding the env var, I can add a conditional for loading the proxy if DISCONNECTED_PROXY is false, and this env var will be true by default and could be overwritten from my workflow. I'll work on it.

Reply:
I have added the `DISCONNECTED_PROXY` env var to enable the proxy load. However, after digging into it, I don't believe it is necessary because only if a previous step creates the proxy-conf.sh will it be loaded in the openshift-qe-network-perf step, and no one other steps that load the proxy-conf make a prior condition.

Reply:
ack - I just wanted some safety here. Thanks!